Dockerfile

```diff
@@ -7,15 +7,21 @@ FROM ${BASE_IMAGE} as builder
 
 # Create the runtime image.
 
-ENV REFRESHED_AT=2022-07-14
-
-LABEL Name="senzing/senzingapi-runtime" \
-      Maintainer="support@senzing.com" \
-      Version="3.1.0"
-
 ARG SENZING_ACCEPT_EULA="I_ACCEPT_THE_SENZING_EULA"
 ARG SENZING_APT_INSTALL_PACKAGE="senzingapi-runtime=3.1.2-22194"
-ARG SENZING_APT_REPOSITORY_URL="https://senzing-production-apt.s3.amazonaws.com/senzingrepo_1.0.0-1_amd64.deb"
+ARG SENZING_APT_REPOSITORY_NAME="senzingrepo_1.0.0-1_amd64.deb"
+ARG SENZING_APT_REPOSITORY_URL="https://senzing-production-apt.s3.amazonaws.com"
+
+ARG IMAGE_NAME="senzing/senzingapi-runtime"
+ARG IMAGE_MAINTAINER="support@senzing.com"
+ARG IMAGE_REFRESHED_AT="2022-07-14"
+ARG IMAGE_VERSION="3.1.1"
+
+ENV REFRESHED_AT=${IMAGE_REFRESHED_AT}
```

Comment:
Until we find some solid literature backing up this technique, can we continue to use:
```
ENV REFRESHED_AT=YYYY-MM-DD
```
Once we find literature supporting a better technique, I'd be happy to follow it.